src/website/views/components/alert.pug

```diff
@@ -13,113 +13,219 @@ h4 Base
 .example.-mb--3
   .-p--1
     each type in ['success', 'danger', 'warning', 'info', 'muted']
-      .m-alert.-bubble.-m--2(class=`-${type}`)
+      .m-alert.-bubble.-m--2.-md(class=`-${type}`)
```

Comment:
Base alert is -md, it should work without the class -md. When we remove this class the alerts get the -sm size which is incorrect. Please change it in all the examples.

Reply:
@janibolkvadze In addition to this, the base alert type is -bubble. So if the bubble class is removed, it should still render as a bubble alert. We have a lot of alerts in production today without the -bubble class that will break with this PR. Here's an example of one on the grid page. Notice the border isn't rendering and the size doesn't match md
![image](https://user-images.githubusercontent.com/130654/67229340-f682a080-f3ef-11e9-8732-5ebeb534e626.png)